website/package.json

```diff
@@ -18,7 +18,6 @@
 		"bootstrap": "^5.2.0",
 		"bootstrap-icons": "^1.9.1",
 		"classnames": "^2.2.6",
-		"html-inline-css-webpack-plugin": "^1.11.1",
```

Comment:
the HTML bundler plugin replace the functionality of the html-inline-css-webpack-plugin